content/en/docs/concepts/services-networking/_index.md

```diff
@@ -7,58 +7,90 @@ description: >
-{{< /note >}}
-
-This model is not only less complex overall, but it is principally compatible
-with the desire for Kubernetes to enable low-friction porting of apps from VMs
-to containers.  If your job previously ran in a VM, your VM had an IP and could
-talk to other VMs in your project.  This is the same basic model.
-
-Kubernetes IP addresses exist at the `Pod` scope - containers within a `Pod`
-share their network namespaces - including their IP address and MAC address.
-This means that containers within a `Pod` can all reach each other's ports on
-`localhost`. This also means that containers within a `Pod` must coordinate port
-usage, but this is no different from processes in a VM.  This is called the
-"IP-per-pod" model.
-
-How this is implemented is a detail of the particular container runtime in use.
-
-It is possible to request ports on the `Node` itself which forward to your `Pod`
-(called host ports), but this is a very niche operation. How that forwarding is
-implemented is also a detail of the container runtime. The `Pod` itself is
-blind to the existence or non-existence of host ports.
-
-Kubernetes networking addresses four concerns:
-- Containers within a Pod [use networking to communicate](/docs/concepts/services-networking/dns-pod-service/) via loopback.
-- Cluster networking provides communication between different Pods.
-- The [Service](/docs/concepts/services-networking/service/) API lets you
-  [expose an application running in Pods](/docs/tutorials/services/connect-applications-service/)
-  to be reachable from outside your cluster.
-  - [Ingress](/docs/concepts/services-networking/ingress/) provides extra functionality
-    specifically for exposing HTTP applications, websites and APIs.
-  - [Gateway API](/docs/concepts/services-networking/gateway/) is an {{<glossary_tooltip text="add-on" term_id="addons">}}
-    that provides an expressive, extensible, and role-oriented family of API kinds for modeling service networking.
-- You can also use Services to
-  [publish services only for consumption inside your cluster](/docs/concepts/services-networking/service-traffic-policy/).
+The Kubernetes network model is built out of several pieces:
+
+  * Each [`Pod`](/docs/concepts/workloads/pods/) in a cluster gets its
+    own unique cluster-wide IP address.
+
+      * A pod has its own private network namespace which is shared by
+        all of the containers within the pod. Processes running in
+        different containers in the same pod can communicate with each
+        other over `localhost`.
+
+  * The pod network (or "cluster network") handles communication
+    between pods:
+
+      * All pods can communicate with all other pods, whether on the
+        same [node](/docs/concepts/architecture/nodes/) or on
+        different nodes, without NAT.
+
+          * (For platforms that support host-network pods (e.g.
+            Linux), this rule applies to host-network pods as well.)
```

Comment:
Aligning to the [style guide](https://kubernetes.io/docs/contribute/style/style-guide/):
```suggestion
      * All pods have a route available to all other pods, whether two Pods are on the
        same [node](/docs/concepts/architecture/nodes/) or on
        different nodes. This is a direct means to communicate (no NAT).

          * (For platforms that support host-network pods (Linux),
          this rule applies to host-network pods as well.)
```
Pods may or may not be able to actually communicate, due to:
- packet filtering (and similar controls)
- misconfiguration, I suppose
- outages

Reply:
I brought back the "barring intentional network segmentation" text from before. That applies to the "agents on a node" subclause too, so I put it at the "the pod network handles communication" level rather than the "all pods can communicate" level.

On the Linux thing, it felt weird to be calling out the Linux behavior, which is the expected behavior, rather than the Windows behavior, which is considered wrong, so I flipped that.